src/types/formatted/phoneNumber.ts

```diff
@@ -2,7 +2,7 @@ import { PublishedParagraph } from '@/types/formatted/publishedEntity'
 
 export type PhoneNumber = PublishedParagraph & {
   extension: string
-  label: string
+  label?: string
```

Comment:
label is optional since the component can also derive the label from phone type

Reply:
One of these days, I'd love to switch on strict mode so these optional properties are actually meaningful. 😮‍💨 

Thanks for being deliberate about this, though!